templates_qc/example_tma_fov_generate.ipynb

```diff
@@ -0,0 +1,430 @@
+{
```

Comment:
<p>This is nice. Before, I would say the notebooks had too little information. Now, they have a bit too much information. We'll find the goldilocks zone next time. The goal is to provide the user enough information to understand what they need to do, but not so much that they get overwhelmed or miss key details. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Sounds good.</p>

Reply:
Per our discussion today, before merging in this branch @ngreenwald will push a commit with the new notebook description at the top of `example_tma_fov_generate.ipynb`.